pkg/component/component.go

```diff
@@ -321,3 +323,34 @@ func Exists(client *occlient.Client, componentName string) (bool, error) {
 	}
 	return false, nil
 }
+
+// Get Component Description
+func GetComponentDesc(client *occlient.Client, cmpnt string) (path string, componentURL string, appStore []storage.StorageInfo, err error) {
```

Comment:
pass `applicationName string` as a parameter, we should not call `application.GetCurrent` inside package component